internal/compiler/fuzz/fuzz_targets/fuzz_compiler.rs

```diff
@@ -0,0 +1,28 @@
+// Copyright © SixtyFPS GmbH <info@slint-ui.com>
+// SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-commercial
+
+#![no_main]
+use libfuzzer_sys::fuzz_target;
+
+use i_slint_compiler::diagnostics::{BuildDiagnostics, SourceFile};
+use i_slint_compiler::parser::{parse_tokens, Token};
+
+fuzz_target!(|tokens: Vec<Token>| {
+    let source_file = SourceFile::default();
+    let mut diags = BuildDiagnostics::default();
+
+    let doc_node = parse_tokens(tokens, source_file, &mut diags);
+
+    let rt = tokio::runtime::Runtime::new().unwrap();
+
+    rt.block_on(async move {
+        let (_, _) = i_slint_compiler::compile_syntax_node(
+            doc_node,
+            diags,
+            i_slint_compiler::CompilerConfiguration::new(
+                i_slint_compiler::generator::OutputFormat::Llr,
```

Comment:
I wonder if it makes sense to perhaps go through the C++/Rust generators as well, if they have stray unwrap calls for example.

Reply:
That probably makes sense, yes.

I wanted to add something to lower LLR later. If we really want to do fuzzing a lot, then we need to look into making more of our structures (and those we use from other crates) compatible with `arbitrary`. Otherwise the whole thing gets much too slow for practical use: We need too much time to generate the inputs then!